containers/docker-ci/Dockerfile

```diff
@@ -66,7 +66,6 @@ RUN patch -p0 < extract.patch && \
     patch -p0 < rpm.patch
 
 WORKDIR /opt/app-root/src
-USER 1001
```

Comment:
Why need to remove the user ?

Reply:
If we have this line, it has no permission to update krb5.conf ("cp openlcs-playbooks/roles/openlcs/files/krb5.conf /etc/").

Reply:
So can we move the cp action into the Dockerfile?